src/avfoundation.cs

```diff
@@ -7612,9 +7619,21 @@ interface AVCaptureVideoPreviewLayer {
 		[Static, Export ("layerWithSession:")]
 		AVCaptureVideoPreviewLayer FromSession (AVCaptureSession session);
 
+		[Static]
 		[Export ("initWithSession:")]
+		IntPtr InitWithConnection (AVCaptureSession session);
+
+		[iOS (8,0), Mac (10,2)]
+		[Static]
+		[Export ("initWithSessionWithNoConnection:")]
+		IntPtr InitWithNoConnection (AVCaptureSession session);
+
+		[Wrap ("this (InitWithConnection (session))")]
 		IntPtr Constructor (AVCaptureSession session);
 
+		[Wrap ("this (withConnection? InitWithConnection (session) : InitWithNoConnection (session))")]
+		IntPtr Constructor (AVCaptureSession session, bool withConnection);
```

Comment:
It would be better to use an enum (in case future SDK have more surprises) as the enum members can be decorated with availability attributes. This API lacks any information about when it's fine to call it.